scripts/main.js

```diff
@@ -22,6 +40,9 @@ document.addEventListener('keyup', (e) => {
     });
 });
 
-
-console.log("Entering game")
-new Game();
\ No newline at end of file
+document.addEventListener('visibilitychange', () => {
```

Comment:
whooaa i didn't know visibilitychange was a thing -- is this to handle when the user switches tabs or moves between screens?

Reply:
this is for switching tabs, unfortunately i dont think theres a way to get the event for switching screens or windows